spec/mailers/candidate_mailer_ucas_match_initial_email_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe CandidateMailer, type: :mailer do
+  include TestHelpers::MailerSetupHelper
+
+  subject(:mailer) { described_class }
+
+  describe '.ucas_match_initial_email_duplicate_applications' do
+    let(:ucas_match) { create(:ucas_match) }
+    let(:application_form) { ucas_match.candidate.application_forms.first }
+    let(:application_choice) { application_form.application_choices.first }
+    let(:email) { mailer.ucas_match_initial_email_duplicate_applications(application_choice) }
+
+    it 'sends an email with the correct subject' do
+      expect(email.subject).to include(I18n.t!('candidate_mailer.ucas_match_initial_email.duplicate_applications.subject'))
+    end
+
+    it 'sends an email with the correct heading' do
+      expect(email.body.encoded).to include("Dear #{application_form.full_name}")
+    end
+
+    it 'sends an email containing the course name and code in the body' do
+      course_name_and_code = application_choice.course.name_and_code
+
+      expect(email.body).to include(course_name_and_code)
+    end
+
+    it 'sends an email containing the candidate full name in the body' do
+      full_name = application_form.full_name
+
+      expect(email.body).to include(full_name)
+    end
+
+    it 'sends an email containing the provider name in the body' do
+      provider_name = application_choice.course.provider.name
+
+      expect(email.body).to include(provider_name)
+    end
+
+    it 'sends an email containing the date that the application needs to be withdrawn by' do
+      withdraw_application_date = 10.business_days.after(Time.zone.today).to_s(:govuk_date)
```

Comment:
It would be good not to repeat the calculation out here — if we could ask `ucas_match` for the date that might be a bit tidier?

Reply:
after discussing this with @AgaDufrat `TimeLimitConfig` looked like a more appropriate place so I moved all rules there and used `TimeLimitCalculator` to retrieve the calculated dates. The relevant changes are in my last commit:  https://github.com/DFE-Digital/apply-for-teacher-training/pull/3463/commits/00b7a0db5e2e5bf35fe85e3f17843bdc0c5ec624